src-vue/src/plugins/store.ts

```diff
@@ -87,13 +88,18 @@ export const store = createStore<FlightCoreStore>({
         checkNorthstarUpdates(state) {
             _get_northstar_version_number(state);
         },
-        toggleDeveloperMode(state) {
+        toggleDeveloperMode(state, affectMenuStyle = false) {
```

Comment:
Not sure how I fell about this as it kinda combines developer and debug mode but on the other side it's still the result as before, so idc too much tbh ^^

Reply:
So that when you change your mind, fixing it will be fast 😄